frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -87,6 +87,23 @@ export interface QuestionResultsReady {
 
 const getResponseField = (i: number): string => (i === 0 ? '$survey_response' : `$survey_response_${i}`)
 
+function duplicateExistingSurvey(survey: Survey | NewSurvey): NewSurvey {
+    const copiedNewSurvey = Object.keys(NEW_SURVEY).reduce((acc, key) => {
+        acc[key] = survey[key]
```

Comment:
actually changed my mind, lets not do this. The problem is if in the future we add an optional field that isn't part of `NEW_SURVEY` , this update will not make it to the duplicate request, which will be wrong.

We discard extra fields anyway so should be ok even if we're sending created_at and created_by